frontend/amundsen_application/static/js/components/Tour/styles.scss

```diff
@@ -1,7 +1,13 @@
 // Copyright Contributors to the Amundsen project.
 // SPDX-License-Identifier: Apache-2.0
 
+@import 'variables';
+
 /* Tour step tooltip content container */
+.react-joyride__tooltip {
+  border-radius: $popover-border-radius !important;
```

Comment:
is it necessary to use `!important` here?

Reply:
Yeah, not ideal but yes.
It is usually OK to do this when dealing with third party component customization though.